Assets/Notero/RaindropGameplay/UI/MidiGameplayConnector.cs

```diff
@@ -0,0 +1,148 @@
+using Notero.RaindropGameplay.UI;
+using Notero.Utilities;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Notero.RaindropGameplay.Core
+{
+    [System.Serializable]
+    public class PrefabInfo
+    {
+        public string Name;
+        public GameObject gamePrefab;
```

Comment:
อันนี้เรื่อง naming นิดหน่อยค่ะ
```suggestion
        public GameObject GamePrefab;
```